resources/tests/test_resource_api.py

```diff
@@ -84,19 +88,57 @@ def test_user_permissions_in_resource_endpoint(api_client, resource_in_unit, use
     assign_perm('unit:can_make_reservations', group, resource_in_unit.unit)
     api_client.force_authenticate(user=user)
     _check_permissions_dict(api_client, resource_in_unit, is_admin=False,
-                            can_make_reservations=True, can_ignore_opening_hours=False)
+                            can_make_reservations=True, can_ignore_opening_hours=False,
+                            can_bypass_payment=False)
     remove_perm('unit:can_make_reservations', group, resource_in_unit.unit)
 
     resource_group = resource_in_unit.groups.create(name='rg1')
     assign_perm('group:can_make_reservations', group, resource_group)
     api_client.force_authenticate(user=user)
     _check_permissions_dict(api_client, resource_in_unit, is_admin=False,
-                            can_make_reservations=True, can_ignore_opening_hours=False)
+                            can_make_reservations=True, can_ignore_opening_hours=False,
+                            can_bypass_payment=False)
 
     assign_perm('unit:can_ignore_opening_hours', group, resource_in_unit.unit)
     api_client.force_authenticate(user=user)
     _check_permissions_dict(api_client, resource_in_unit, is_admin=False,
-                            can_make_reservations=True, can_ignore_opening_hours=True)
+                            can_make_reservations=True, can_ignore_opening_hours=True,
+                            can_bypass_payment=False)
+
+    # user has explicit permission to bypass payment
```

Comment:
This comment seems to be wrong?

Reply:
What do you mean? Do you mean it should be `group has explicit permission`?